system/Common.php

```diff
@@ -683,22 +683,14 @@ function helper($filenames)
      *
      * @return bool
      *
-     * @codeCoverageIgnore Cannot be tested fully as PHPUnit always run in CLI
+     * @codeCoverageIgnore Cannot be tested fully as PHPUnit always run in php-cli
      */
     function is_cli(): bool
     {
-        if (PHP_SAPI === 'cli') {
-            return true;
-        }
-
```

Comment:
This is probably the most common setting. So I'm just wondering why we're removing this... were there any issues with it?

Reply:
Oh... okay sorry... I missed your description. I will test it tomorrow but still... I have mixed feelings about removing this check as I don't think it was returning any false results and it's the most common setting we can rely on.

Reply:
From a bit of research from php.net, it seems the `STDIN` is defined by the CLI SAPI. So we can at least be sure that if `STDIN` is defined then we're running in php-cli.